doc/release-notes-1.3.0.rst

```diff
@@ -0,0 +1,571 @@
+.. _ncs_release_notes_130:
+
+|NCS| v1.3.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.0 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* Support for Thread and Zigbee
+* Support for Bluetooth Mesh
+* New nRF9160 samples and libraries
+* Application update through external flash
+* General updates and bugfixes
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.3.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+
+Supported modem firmware
+************************
+
+This version of the |NCS| supports the following modem firmware for cellular IoT applications:
+
+* mfw_nrf9160_1.2.0
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+nRF9160
+=======
+
+* Added:
+
+  * :ref:`lib_date_time` library - maintains the current date-time UTC from multiple time sources (modem, NTP servers).
+  * :ref:`lib_ftp_client` library - can be used to download or upload FTP server files.
+  * :ref:`lib_nrf_cloud_agps` library - provides a way to request and process A-GPS data from nRF Cloud to be used with the nRF9160 SiP.
+    The :ref:`nrf_cloud_agps_sample` sample demonstrates how to use the library.
+  * :ref:`connectivity_bridge` application with dual UART and Bluetooth LE support - replaces the :ref:`usb_uart_bridge_sample` sample.
+
+* :ref:`gps_with_supl_support_sample` sample: (?)
```

Comment:
https://github.com/nrfconnect/sdk-nrf/tree/master/samples/nrf9160/gps

Reply:
That is where the link currently points.